lib/scrabble/scrabble.rb

```diff
@@ -0,0 +1,119 @@
+module ScrabbleSinatra
+  class ScrabbleSinatra
```

Comment:
Although we have done it previously - I would advise against using the same name for a module and a class